custom/ewsghana/tasks.py

```diff
@@ -33,12 +37,18 @@
                   526, 4, 30, 1, 14, 23, 521, 532, 516, 461, 520, 525, 961, 641, 257, 348]
 
 
-# @periodic_task(run_every=timedelta(days=1), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
+@periodic_task(run_every=crontab(hour="23", minute="55", day_of_week="*"),
+               queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
```

Comment:
(i see we would first have to enable steady sync for this to do much so it should be safe as long as we don't enable it for many - possibly any - domains yet)